frontend/openchat-client/src/state/chat_details/server.svelte.ts

```diff
@@ -26,6 +27,7 @@ export class ChatDetailsServerState {
     #rules = $state<VersionedRules>(emptyRules());
     #bots = $state<ReadonlyMap<string, ExternalBotPermissions>>(new Map());
     #apiKeys = $state<ReadonlyMap<string, PublicApiKeyDetails>>(new Map());
+    #webhooks = $state<WebhookDetails[]>([]);
```

Comment:
Given that these are keyed we might want to consider storing them in a map so we can easily look them up. Doesn't make them any harder to iterate really.